.github/workflows/test.yml

```diff
@@ -0,0 +1,28 @@
+name: Test
+
+on: [push, pull_request]
```

Comment:
Not sure if this should only trigger on `pull_request` rather than `[push, pull_request]`? also will this replace the current travis ci setup? please advice here

Reply:
would probably replace travis, both push and PR is ok i think? Maybe you just need `push`?

Reply:
I removed the `push` and tested `pull_request` by itself and it worked on 

- Opening PRs
- Syncing or pushing changes to the PR
- Re-opining the PR

Noticed, when I had the `push, pull_request` it triggers two unnecessary jobs 

Release job will only trigger when anything gets pushed to `master`